src/edu/stanford/nlp/pipeline/demo/corenlp

```diff
@@ -22,23 +22,37 @@
 #
 SERVER_USER="nlp"
 CORENLP_DIR="/opt/corenlp"
+PROP_FILE="$CORENLP_DIR/server.properties"
+LISTEN_PORT=9000
+JVM_MAX_MEMORY=8g
 
 
 do_start()
 {
   if [ -e "$CORENLP_DIR/corenlp.shutdown" ]; then
     echo "CoreNLP server is already running!"
-    echo "If you are sure this is a mistake, delete the file:"
-    echo "$CORENLP_DIR/corenlp.shutdown"
+    # echo "If you are sure this is a mistake, delete the file:"
```

Comment:
Better remove those two lines, instead of commenting out. (You're using Git.)

Reply:
fixed